docs/docs/terminology.md

```diff
@@ -57,14 +56,14 @@ A **collection** is a set of configuration files that can be committed to a vers
 * Connector configuration
 * Snapshot configuration
 * Test file
-* Optional Exclusion file for a specific resource, or a specific test or a combination of both.
+* Optional Exclusion file for specific resource/specific test/combination of both.
 
 The **Collection** is a powerful concept that can be used based on the organization's definition. It could work as a project, a repo, a business sector, or a company.
 
 These files should follow your development project that would usually contain **infrastructure as code** files such as:
 
 * **AWS CloudFormation** templates
-* **Azure ARM** templates
+* **AWS ARM** templates
```

Comment:
`Azure ARM` template is the correct one. we don't have AWS ARM

Reply:
I have made the modification.